### PR TITLE
Added bloom, brightness and ssao settings

### PR DESCRIPTION
**Pull request by dipokDipu**
## Summary by CodeRabbit

- **New Features**
  - Added new video settings for brightness, bloom, and ambient occlusion in Unity projects using the Universal Render Pipeline (URP).
  - Introduced a comprehensive video settings package with installation instructions and usage guidelines.
  - Enhanced scene profiles with updated settings for bloom, vignette, and color adjustments.
  - Implemented a functional example with UI elements for adjusting video settings in real-time.

- **Documentation**
  - Updated README and CHANGELOG with the latest features and instructions for users.

- **Enhancements**
  - Improved shader options for TextMesh Pro, including new shaders for mobile platforms and various text rendering effects.
  - Added assets and prefabs for better management of video settings and UI elements.

### CODE REVIEW

File: Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/DisplayUI.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+using System.Collections.Generic;
+using Studio23.SS2.Settings.Video.Core;
+using UnityEngine;
+
+namespace Studio23.SS2.Settings.Video.Samples
+{
+    public class DisplayUI : MonoBehaviour
+    {
+        [SerializeField] private LoopedStepper _Resolutionstepper;
```
Comment (zubEpty) [inline Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/DisplayUI.cs:10]: variable naming incorrect

File: Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/LoopedStepper.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        {
+            ShowText();
+            if (!_isLive) return;
+            ApplyAction();
+        }
+
+        private void ApplyAction()
+        {
+            SelectedIndexUpdated?.Invoke(_selectedIndex);
+        }
```
Comment (BDeshiDev) [inline Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/LoopedStepper.cs:78]: apply action name should be renamed to reflect that invokes SelectedIndexUpdated event.

File: Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/LoopedStepper.cs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+using TMPro;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace Studio23.SS2.Settings.Video.Samples
+{
+    public class LoopedStepper : MonoBehaviour
+    {
+        [SerializeField] private TextMeshProUGUI DisplayText;
```
Comment (BDeshiDev) [inline Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/LoopedStepper.cs:12]: naming conventio

File: Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/LoopedStepper.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            }
+            else
+            {
+                _selectedIndex = _data.Count - 1;
+            }
+
+            Apply();
+        }
+
+        public void Apply(bool checkLiveStatus = true)
```
Comment (BDeshiDev) [inline Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/LoopedStepper.cs:68]: checkLiveStatus is not used.

File: Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/LoopedStepper.cs [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        {
+            ShowText();
+            if (!_isLive) return;
```
Comment (BDeshiDev) [inline Assets/Samples/Video Settings System/1.0.1/Functional Example/Scripts/LoopedStepper.cs:71]: IsLive should be renamed to reflect that it controls whether the event is invoked.

**Review by BDeshiDev — CHANGES_REQUESTED** [review]

**Review by Warhammer4000 — APPROVED** [review]

**Pull request closed without merging**